churaverse-plugins-client/src/playerPlugin/errors/playerRendererNotFoundError.ts

```diff
@@ -0,0 +1,7 @@
+import { PlayerPluginError } from './playerPluginError'
+
+export class PlayerRendererNotFoundError extends PlayerPluginError {
+  public constructor() {
+    super('playerRendererが見つかりませんでした。')
+  }
+}
```

Comment:
`PlayerRendererNotFoundError`が実行されるのは、特定のplayerIdのPlayerRendererを参照した際にundefinedになる場合だと思います。
そのため、以下のようなエラーメッセージに変更した方が良いかと思います。

```suggestion
export class PlayerRendererNotFoundError extends PlayerPluginError {
   public constructor(playerId: string) {
     super(`playerId: ${playerId} の playerRenderer が存在しない`)
   }
 }
```